pom.xml

```diff
@@ -15,6 +15,10 @@
     </repository>
   </repositories>
 
+  <properties>
+    <jenkins.version>1.642.3</jenkins.version>
```

Comment:
This would be the new minimum version. This property appears to be used in the parent pom for dependencies (the `org.jenkins-ci.plugins:plugin:2.25` version).

The updated dependency on `workflow-step-api` requires the Jenkins version as the base, and the tests fail with lots of classloading errors if at least this version isn't used.

There _may_ be some hackery that can done to avoid bringing in the new workflow API version for https://github.com/jenkinsci/ansicolor-plugin/pull/109/files#diff-8c1fae68c810376b2cd62d70313c4c3cR72 , but I am not confident in my Jenkins plugin development knowledge to make a decision on this stuff.